doc/deps.md

```diff
@@ -106,21 +135,13 @@ cd thrift-0.13.0
 
 # Ant is required for having Java support in Thrift.
 sudo apt-get install ant
-
-# Thrift can generate stubs for many programming languages. The configure
-# script looks at the development environment and if it finds the environment
-# for a given language then it'll use it. For example in the previous step npm
-# was installed which requires NodeJS. If NodeJS can be found on your machine
-# then the corresponding stub will also compile. If you don't need it then you
-# can turn it off: ./configure --without-nodejs.
-#
-# In certain cases, installation may fail if development libraries for
-# languages are not installed on the target machine. E.g. if Python is
-# installed but the Python development headers are not, Thrift will unable to
-# install. Python, PHP and such other Thrift builds are NOT required by
-# CodeCompass.
-
-./configure --prefix=<thrift_install_dir> --with-python=NO --with-php=NO
+./configure --prefix=<thrift_install_dir> --silent --without-python \
+  --enable-libtool-lock --enable-tutorial=no --enable-tests=no      \
+  --with-libevent --with-zlib --without-nodejs --without-lua        \
+  --without-ruby --without-csharp --without-erlang --without-perl   \
+  --without-php --without-php_extension --without-dart              \
+  --without-haskell --without-go --without-rs --without-haxe        \
+  --without-dotnetcore --without-d --without-qt4 --without-qt5;     \
```

Comment:
Is this command terminated properly? `;` and then a `\`

Reply:
No, I just copied it from the Dockerfile, and forgot to remove the last \.